packages/blade/package.json

```diff
@@ -242,6 +242,7 @@
   },
   "peerDependencies": {
     "@table-library/react-table-library": "4.1.7",
+    "@emotion/react": "11.11.1",
```

Comment:
Another thing we should do that we didn't on other peerDeps, is instead of pinning them we should use ranges, otherwise consumers will have to use the exact version down the the patch otherwise there will be mismatches.  

```
@emotion/react: ^11.11.1
```

Reply:
Yup, done!